docs/book/src/getting-started.md

```diff
@@ -0,0 +1,537 @@
+
+<aside class="note">
+<h1>RBAC generate under config/rbac</h1>
+
+For each Kind generate Kubebuilder will either scaffold rules with view and edit permissions. (i.e. `memcached_editor_role.yaml` and `memcached_viewer_role.yaml`)
+Those rules are not applied on the cluster when you deploy your solution with `make deploy IMG=myregistery/example:1.0.0`.
+Those rules are aimed to help our system admins to allow them grant permissions to group of users.
+
+</aside>
+
+### Manager (main.go)
+
+The [Manager][manager] plays a crucial role in overseeing Controllers, which in turn enable operations on the cluster side. 
+If you inspect the `cmd/main.go` file, you'll come across the following:
+
+```go
+...
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:                 scheme,
+    Metrics:                metricsserver.Options{BindAddress: metricsAddr},
+    HealthProbeBindAddress: probeAddr,
+    LeaderElection:         enableLeaderElection,
+    LeaderElectionID:       "1836d577.testproject.org",
+    // LeaderElectionReleaseOnCancel defines if the leader should step down voluntarily
+    // when the Manager ends. This requires the binary to immediately end when the
+    // Manager is stopped, otherwise, this setting is unsafe. Setting this significantly
+    // speeds up voluntary leader transitions as the new leader don't have to wait
+    // LeaseDuration time first.
+    //
+    // In the default scaffold provided, the program ends immediately after
+    // the manager stops, so would be fine to enable this option. However,
+    // if you are doing or is intended to do any operation such as perform cleanups
+    // after the manager stops then its usage might be unsafe.
+    // LeaderElectionReleaseOnCancel: true,
+})
+if err != nil {
+    setupLog.Error(err, "unable to start manager")
+    os.Exit(1)
+}
+```
+
+The code snippet above outlines the configuration [options][options-manager] for the Manager. While we won't be altering this in our current example, 
+it's crucial to understand its location and the initialization process of your operator-based image. The Manager is responsible for overseeing the controllers 
+that are produced for your operator's APIs.
+
+### Checking the Project running in the cluster
+
+At this point, you can primarily execute the commands highlighted in the [quick-start][quick-start]. 
+By executing `make build IMG=myregistry/example:1.0.0`, you'll build the image for your project. For testing purposes, it's recommended to publish this image to a 
+public registry. This ensures easy accessibility, eliminating the need for additional configurations. Once that's done, you can deploy the image 
+to the cluster using the `make deploy IMG=myregistry/example:1.0.0` command.
+
+## Next Steps
+
+- To delve deeper into developing your solution, consider going through the provided tutorials.
+- For insights on optimizing your approach, refer to the [Best Practices][best-practices] documentation.
+
```

Comment:
Add a link for installation guid

Reply:
Here we are adding the Next Steps.
Ideally who is in this point already followed up the Quick Start and this overview, their for the person has the Kubebuilder CLI installed.

Reply:
Got it.